src/plone/restapi/tests/test_query_parsers.py

```diff
@@ -42,6 +42,10 @@ def test_casts_complex_query_values_to_string(self):
         qp = ZCTextIndexQueryParser()
         self.assertEqual({"query": "42"}, qp.parse({"query": 42}))
 
+    def test_strip_parenthesis_from_value(self):
+        qp = ZCTextIndexQueryParser()
+        self.assertEqual("foo bar", qp.parse("foo (bar)"))
```

Comment:
Nitpicking: maybe add the other parenthesis here as well. Just to document that it works for all of them. A dev who reads this test might wonder about it. Just two more lines :)